CHANGES.txt

```diff
@@ -2,6 +2,151 @@
+* fix broken image links on docs/source/user_manual/quickstart.rst (#588)
+* Replace handmade api docs with autogenerated api docs (#637)
+* Miscellaneous documentation fixes/changes (#642)
+* Add module doc string for chaco.api (#649, #665)
+* Use #: instead of # when needed (#650)
+* Update modules_and_classes.rst (#652)
+* Remove annotated examples from the usermanual (#653)
+* Use sphinx todo for currently empty sections of docs (#654)
+* Remove optparse from example (#655)
+* Delete the quickstart page from the docs (#668)
+* Remove references to chaco.shell from the documentation (#669)
+* Update van der waal example (#680)
+* Remove scipy tutorial content (#681)
+* Remove chaco_talk_apr_2008 from docs (#682)
+* Update hyetopgraph example (#687)
+* Remove unused related images from docs (#712)
+* Organize annotated examples (#721)
+* Link to "how do i", not "faq", from home page (#722)
+* Matplotlib to chaco cheatsheet in docs (#723)
+
+Maintenance
+
+* Import from traits.api instead of from traits.has_traits (#509)
+* Remove commented code (#511, #556, #582, #647)
+* Drop Python < 3.6 support (#554, #555)
+* Remove redundant 'control' in ZoomTool (#565)
+* Clean up style, fix warnings and broken links in tutorials/examples/faq (#568)
+* Update copyright year in license.txt (#569)
+* Remove EOF file markers (#579)
+* Remove unnecessary unittest.main calls in tests (#581)
+* Replace on_trait_change with observe (#585, #590, #591, #625, #630)
+* Remove dummy return statements (#592, #593, #594)
+* Run black on the codebase (#599)
+* Remove outdated coverage badge (#623)
+* Use requires_redraw when possible (#626)
+* Deal with __getstate__ methods (#633)
+* Dont use traitsui.api as tui (#634)
+* Use api modules for ETS packages where possible (#643)
+* Remove unnecessary print statements (#646)
+* Use @property decorator instead of property function (#651)
+* Its TraitsUI, not Traits UI (#656)
+* Remove unnecessary uses of "use_draw_order" (#661)
+* Remove all uses/definitions of use_draw_order (#671)
+* Import color palette from chaco.api directly (#675)
+* Remove backwards compatibily imports from enable (#679)
+* Move tests in "tests_with_backend" submodule into "tests" submodule (#683)
+* Rename test so it is discovered by unittest discover (#693)
+* Cleanup License files (#696)
+* Update "super" usage (#703, #706, #725)
+* Use a plot default method instead of overriding the init method in examples (#704, #709)
+* Replace custom "assert_raises" with "unittest.TestCase.assertRaises" (#705)
+* Concatenate change logs (#710)
+* re-export PolarLineRenderer and PolarMapper from chaco.api (#732)
+* add editable option to ci install command (#733)
+* remove calls to Component._draw_component (#737)
+* Add new chaco.plots submodule (#738)
+* use mock through unittest (#740)
+* Consolidate overlays into "chaco.overlays" (#741)
+* Replace bg_color with bgcolor (#742)
+* Revive ScalyPlot (#744)
+
+Removals/Deprecations
```

Comment:
I think this section needs to be moved to right below "Features/Enhancements/Improvements" - very very important given the large volume of code killed